### PR TITLE
Logic control modules added

### PR DESCRIPTION
**Pull request by JayKowski**
-Added Logic modules to control the mathematical operations of the calculator app. They are however not yet hooked up to work with the UI

### CODE REVIEW

File: src/logic/calculate.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
+/* eslint-disable */
```
Comment (Forison) [inline src/logic/calculate.js]: do not disable linter else you miss the key learning objective.

File: src/logic/operate.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
+/* eslint-disable */
```
Comment (Forison) [inline src/logic/operate.js]: do not disable linter else you miss the key learning objective.

File: src/logic/operate.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+import Big from 'big.js';
+
+const operate = (numberOne, numberTwo, operation) => {
+  let res = 0;
```
Comment (Forison) [inline src/logic/operate.js:8]: **[OPTIONAL]**
We could wrap `numberOne` and `numberTwo` in big.js and store them in a new variable one time. So we can keep calling them in the various conditional block.

**Review by Forison — CHANGES_REQUESTED** [review]
## Status:: Changes required❌
# 🙋‍♂️ @JayKowski 
#### You have done a good job on this one so far, however, there are a few issues we have to address to get his project approved👨.
* [ ] Pull Request has a meaningful title and a brief description ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/descriptive_prs.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))
* [ ]  Do not make any changes in config files - they represent style guidelines that you share with your team - which is a group of all Microverse students. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/linter_config.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/54883773/))
- Check my comment below👇.
### That is it, kindly make the change and submit a re-review👨‍🔧
#### Happy coding👨‍💻 
Addo Forison

File: src/logic/operate.js [hunk 4]
```diff
@@ -1,18 +1,21 @@
...
+    res = (fNum + sNum).toString();
   } else if (operation === '-') {
-    res = Big(numberOne - numberTwo).toString();
+    res = (fNum - sNum).toString();
   } else if (operation === 'X') {
-    res = Big(numberOne * numberTwo).toString();
+    res = (fNum * sNum).toString();
   } else if (operation === '/') {
-    res = Big(numberOne / numberTwo).toString();
+    res = (fNum / sNum).toString();
```
Comment (shloch) [inline src/logic/operate.js]: dont forget to manage DIVISION BY ZERO error on the next milestone, else it will generate an error

**Review by shloch — APPROVED** [review]
## Hello @JayKowski 

Great job with bringing the corrections above
accomplished

We're ready for the next milestone
Keep up with the good work
Happy coding

**Pull request merged**